tools/upgrade-to-1.1.0.sh

```diff
@@ -0,0 +1,212 @@
+        exit 13 
+    fi
+}
+
+backup() {
+    backup_file_name=backup_${CURRENT_TIME}.yaml
+    echo "--------------------------------------------------------------"
+    echo "taking backup of existing operator installation manifest files"
+    echo "backup namespace"
+    echo "---" >> ${backup_file_name}
+    kubectl get namespace ${current_namespace} -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup serviceaccount details"
+    kubectl get serviceaccount ${current_namespace} -n splunk-operator -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any role defined for splunk operator"
+    kubectl get role splunk:operator:namespace-manager -n ${current_namespace} -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any role-biding defined for splunk operator"
+    kubectl get rolebinding splunk:operator:namespace-manager -n ${current_namespace}  -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any cluster-role defined for splunk operator"
+    kubectl get clusterrole splunk:operator:resource-manager -o yaml  >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup if there are any cluster-role-binding defined for splunk operator"
+    kubectl get clusterrolebinding splunk:operator:resource-manager -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "backup deployment details"
+    kubectl get deployment splunk-operator -n ${current_namespace} -o yaml >> ${backup_file_name}
+    if [ $? == 0 ] 
+    then 
+        echo "" >> ${backup_file_name}
+        echo "---" >> ${backup_file_name}
+    fi
+    echo "--------------------------------------------------------------"
+    echo "backup of all the previsou splunk operator installation is complete, backup file is found in current diretory ${backup_file_name}"
+}
+
+delete_operator() {
+    echo "--------------------------------------------------------------"
+    echo "deleting all the previsous splunk operator resources....."
+    echo "deleting clusterrole"
+    kubectl delete clusterrole splunk:operator:resource-manager 
+    echo "deleting cluster rolebinding"
+    kubectl delete clusterrolebinding splunk:operator:resource-manager 
+    echo "deletign deployment"
+    kubectl delete deployment splunk-operator -n ${current_namespace}
```

Comment:
Doesn't it need to be the first one in the list?

Reply:
fixed